comixed-webui/src/assets/i18n/pt/app.json

```diff
@@ -56,21 +56,26 @@
   "home": {
     "label": {
       "character": "Personagem",
-      "comics-by-character": "Comics Por Personagem",
-      "comics-by-location": "Comics Por Localização",
-      "comics-by-publisher": "Comics Por Editora",
-      "comics-by-series": "Comics Por Série",
-      "comics-by-story": "Comics Por Estória",
-      "comics-by-team": "Comics Por Equipa",
-      "comic-count": "# de Comics",
+      "comic-state-added": "Added",
+      "comic-state-changed": "Changed",
+      "comic-state-deleted": "Deleted",
+      "comic-state-stable": "Stable",
+      "comic-state-unprocessed": "Unprocessed",
       "location": "Localização",
       "publisher": "Editora",
+      "select-collection-type": "Collection Type",
       "series": "Série",
       "story": "Estória",
       "team": "Equipa"
     },
+    "legend": {
+      "collections-top-entries-x-axis": "Number Of Comics",
+      "comic-state-x-axis": "Number Of Comics In Each State"
+    },
     "tab-title": "Início",
     "title": {
+      "comic-collections-chart": "Top Collection Entries",
+      "comic-state-chart": "Comics By State",
       "comic-statistics": "Estatísticas da Biblioteca de Comics",
       "loading-comics": "Carregando Biblioteca..."
```

Comment:
If you want you can add the portuguese translation also

"comic-state-added": "Adicionados",
"comic-state-changed": "Alterados",
"comic-state-deleted": "Apagados",
"comic-state-stable": "Estáveis",
"comic-state-unprocessed": "Por Processar",
"select-collection-type": "Tipo De Coleção",

"collections-top-entries-x-axis": "Quantidade De Comics",
"comic-state-x-axis": "Quantidade De Comics Por Estado"

"comic-collections-chart": "Principais Entradas Da Coleção",
"comic-state-chart": "Comics Por Estado",

Reply:
Thanks. Though I'm going to leave translations until the 1.1 release. That way we keep the translations all together.